winbuild/MakefileBuild.vc

```diff
@@ -262,8 +262,8 @@ ZLIB_LIBS   = zlib.lib
 USE_ZLIB    = true
 ZLIB        = dll
 !ELSEIF "$(WITH_ZLIB)"=="static"
-!IF EXISTS("$(ZLIB_LIB_DIR)\zlibstat.lib")
-ZLIB_LIBS   = zlibstat.lib

```

Comment:
Not a typo, the visual studio zlibstat project's output is [zlibstat.lib](https://github.com/madler/zlib/blob/04f42ceca40f73e2978b50e93806c2a18c1281fc/contrib/vstudio/vc10/zlibstat.vcxproj#L182). Where are you getting zlibstatic.lib from? Are you using a different build method?